dotcom-rendering/src/client/userFeatures/user-features.ts

```diff
@@ -0,0 +1,170 @@
+// This file was migrated from:
+// https://github.com/guardian/commercial/blob/1a429d6be05657f20df4ca909df7d01a5c3d7402/src/lib/user-features.ts
```

Comment:
Thanks for this. You can use [JSDoc `@file`](https://jsdoc.app/tags-file) for these kinds of comments.

Would it be possible to add a similar comment to `user-features-lib.ts` to help future travellers find out about the history of these files?

Reply:
Thanks have amended and updated 👍